packages/xstate-inspect/src/browser.ts

```diff
@@ -94,7 +94,7 @@ export function inspect(
     return undefined;
   }
 
-  const inspectMachine = createInspectMachine(devTools);
+  const inspectMachine = createInspectMachine(devTools, options ?? {});
```

Comment:
nit: probably better to handle this as a default param within `createInspectMachine` since it doesn't really require any options